src/EditorFeatures/VisualBasic/Outlining/Outliners/AccessorDeclarationOutliner.vb

```diff
@@ -34,7 +17,7 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.Outlining
                     spans.Add(
                     VisualBasicOutliningHelpers.CreateRegionFromBlock(
                         methodBlock,
-                        GetBannerText(accessorDeclaration),
+                        accessorDeclaration.ConvertToSingleLine().ToString() & " " & Ellipsis,
```

Comment:
Would like a helper that does this work so it isn't repeated across all outliners.